webapp/components/test-file-results.js

```diff
@@ -315,9 +315,18 @@ class TestFileResults extends WPTFlags(LoadingState(PathInfo(
     } else {
       this.subtestRowCount = 0;
     }
-
+    this._fireEvent('getSubtestRows', { rows });
```

Comment:
```suggestion
    this._fireEvent('subtestrows', { rows });
```
Should be a noun and all lowercase

Reply:
done 👍